src/riak_kv_ts_error_msgs.hrl

```diff
@@ -110,7 +110,48 @@
 ).
 
 -define(
-    E_MISSING_COL_IN_GROUP_BY(FieldName, TableName),
-    iolist_to_binary(io_lib:format(
-        "Error in group by clause, column '~ts' does not exist in table ~ts", [FieldName, TableName]))
-).
\ No newline at end of file
+   E_CANNOT_HAVE_ORDERBY_WITH_INVERSE_DIST_FUNCTION,
+   <<"Inverse distribution functions cannot be used with any of ORDER BY, LIMIT or OFFSET clauses.">>
```

Comment:
Are we confident everyone using an inverse distribution function will actually know what that phrase means?